js/mojs-player.babel.js

```diff
@@ -400,6 +400,12 @@ class MojsPlayer extends Module {
   */
   _onHideStateChange ( isHidden ) {
     this._props.isHidden = isHidden;
+    const { onToggleHide } = this._o;
+    // skip running callback on init rendering
+    if ( this._hideCount && onToggleHide) {
```

Comment:
Why did you choose to check the `this._hideCount` on truthiness? My concern is that it won't fire on the initial render that way?